djclick/test/test_params.py

```diff
@@ -18,17 +18,37 @@ def test_modelinstance_init():
 
 
 @pytest.mark.django_db
-def test_convert_ok(call_command):
+@pytest.mark.parametrize(
+    ('arg', 'value'),
+    (
+        ('--pk', '99'),
+        ('--slug', 'test'),
+        ('--endswith', 'st'),
+    )
+)
+def test_convert_ok(call_command, arg, value):
     from testapp.models import DummyModel
 
-    DummyModel.objects.create()
-    assert call_command('modelcmd', '1').stdout == b'1'
+    DummyModel.objects.create(slug='test')
+    expected = b'<DummyModel: 1>'
+
+    assert call_command('modelcmd', '--pk', '1').stdout == expected
```

Comment:
This is a bit fragile (e.g. when using `--reuse-db`), as the ID could change across runs. Would be better to use `instance = DummyModel.objects.create(slug='test')` and `instance.pk`.